tests/test_datasets.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+import pandas as pd
+import torchvision.transforms as T
+from quickvision.datasets.classification import CSVDataset
+
+df = pd.read_csv('tests/assets/csv_dataset.csv')
+data_dir = 'tests/assets/'
+tfms = T.Compose([
+    T.ToTensor(),
+])
+
+
+class DatasetsTester(unittest.TestCase):
+    def test_csv_dataset(self):
+        flag = False
+        complete_dataset = CSVDataset(df, data_dir, 'Image', 'Label', tfms, 'png')
+        complete_dataset[0]
+        flag = True
+        self.assertTrue(flag)
```

Comment:
Hmm, can you do a `sanity_fit` for CNN model by creating `Dataloader` ?
`sanity_fit` is very efficient, it does not calculate gradients and is fast to test over CPU.
Datasets have to be compatible with our models.

Reply:
I am doing `train_sanity_fit` instead of `sanity_fit`, since I only took two images to test this.